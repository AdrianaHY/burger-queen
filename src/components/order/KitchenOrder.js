import React, { Component } from 'react';

import restaurant from '../../assets/restaurant.svg';

import './KitchenOrder.scss';

class KitchenOrder extends Component {
	render() {
		const { order } = this.props;
		return (
			<div className="kitchen-order-container">
				<div className="text-container">
					<img src={restaurant} className="text-image" alt="restaurant"/>
					<h3>Table 1</h3>
				</div>
				<div className="kitchen-items-container">
					<div className="kitchen-item">
						{ order.map((item) => (
							<p className="kitchen-item-text" key={item}>{item}</p>
						))}
					</div>
					<button className="order-ready">order ready</button>
				</div>
			</div>
		)
	}
}

export default KitchenOrder;