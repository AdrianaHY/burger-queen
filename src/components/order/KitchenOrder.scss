@import '../../styles/_variables.scss';

.kitchen-order-container {
	border-bottom: 2px solid $orange;
	padding: $paddingLarge;

	.text-container {
		display: flex;
		flex-direction: row;
		align-items: center;
		font-family: $fontFamily;
		font-size: $fontSizeHuge;
	}

	.text-image {
		height: 30px;
		padding-right: $paddingMedium;
	}

	.kitchen-items-container {
		display: flex;
		flex-direction: row;
		align-items: center;
		justify-content: space-evenly;
	}

	.kitchen-item {
		background: $lightYellow;
		width: 50vw;
		margin: 0;
	}

	.kitchen-item-text {
		border-bottom: 1px solid $purple;
		font-family: $fontFamily;
		font-size: $fontSizeHuge;
		text-align: left;
		padding: $paddingMedium $paddingLarge;
		margin: 0;
	}

	.order-ready {
		background: transparent;
		border: 1px solid $purple;
		border-radius: 5%;
		color: $purple;
		font-family: $fontFamily;
		font-size: $fontSizeHuge;
		padding: $paddingMedium;
	}
}