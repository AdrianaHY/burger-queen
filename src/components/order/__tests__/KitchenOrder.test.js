import React from 'react';
import { shallow } from 'enzyme';
import KitchenOrder from '../KitchenOrder';

describe('components | order | KitchenOrder', () => {
	it('Renders the component', () => {
		const props = {
			order: [
				'sandwich',
				'coffee',
				'juice'
			]
		};

		const component = shallow(
			<KitchenOrder {...props} />
		);

		expect(component).toMatchSnapshot();
	})
})