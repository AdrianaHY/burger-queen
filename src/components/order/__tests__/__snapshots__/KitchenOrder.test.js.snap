// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | order | KitchenOrder Renders the component 1`] = `ShallowWrapper {}`;
